synapse/lib/cell.py

```diff
@@ -774,6 +774,13 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'type': 'string',
             'hideconf': True,
         },
```

Comment:
Should we add a devops task discussing this?

Reply:
Yeah that would be a good idea